src/guess/stringIndices.js

```diff
@@ -0,0 +1,10 @@
+function stringIndices( input , position ) {
+  let onlyLetters = input.replace(/[!@#$%^&*,]/g, ""),
+      splitString = onlyLetters.split(" ")
```

Comment:
I'm not a fan of the aligned indent of variable declarations - it adds unnecessary work to keep any successive code changed properly aligned.